gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-file-upload/api-documentation-v4-file-upload.component.html

```diff
@@ -23,22 +23,23 @@ <h4>Upload a file</h4>
   </div>
 </div>
 <div class="content__file">
-  <gio-form-file-picker accept=".md,.markdown,.txt" [ngModel]="filePickerValue" (ngModelChange)="onImportFile($event)">
+  <gio-form-file-picker [accept]="getFileFormats()" [ngModel]="filePickerValue" (ngModelChange)="onImportFile($event)">
     <gio-form-file-picker-add-button class="content__file__add-btn">
       <div class="content__file__add-btn__wrapper">
         <mat-icon svgIcon="gio:upload" class="content__file__add-btn__wrapper__icon"></mat-icon>
         <p class="mat-body-strong">
           Drag and drop a file to upload it.<br />
           Alternatively, click here to choose a file.
         </p>
-        <p>Supported file formats: {{ allowedFileExtensions }}</p>
+        <p>Supported file formats: {{ getFileFormats() }}</p>
```

Comment:
Same thing here, should be the same after OnInit, so we can set a variable instead of a function.